storm-server/src/main/java/org/apache/storm/daemon/drpc/RequestFactory.java

```diff
@@ -21,5 +21,6 @@
 import org.apache.storm.generated.DRPCRequest;
 
 public interface RequestFactory<T extends OutstandingRequest> {
-    public T mkRequest(String function, DRPCRequest req);
+
+    T mkRequest(String function, DRPCRequest req);
```

Comment:
Is there a checkstyle policy that requires changes like these?

Reply:
I'm 99.9% percent sure. I could verify if you want. In case it isn't I would make it a check because it's a redundant modifier.

Reply:
OK the first time I missed that `public` was removed as redundant. I had thought only a new line was added.

Reply:
It's not required. It makes sense to remove these redundant modifiers. I'll open a separate PR for that.